tests/listobjects/listobjects.go

```diff
@@ -133,11 +142,13 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 					var streamedObjectIds []string
 
 					clientStream, err := client.StreamedListObjects(ctx, &pb.StreamedListObjectsRequest{
-						StoreId:          storeID,
-						Type:             assertion.Request.Type,
-						Relation:         assertion.Request.Relation,
-						User:             assertion.Request.User,
-						ContextualTuples: assertion.Request.ContextualTuples,
+						StoreId:  storeID,
+						Type:     assertion.Request.Type,
+						Relation: assertion.Request.Relation,
+						User:     assertion.Request.User,
+						ContextualTuples: &pb.ContextualTupleKeys{
+							TupleKeys: assertion.ContextualTuples,
+						},
```

Comment:
```suggestion
						ContextualTuples: contextualTuples,
```

Reply:
removed contextualTuples so we need to stick with &pb.ContextualTupleKeys